src/honey_badger/epoch_state.rs

```diff
@@ -193,6 +196,15 @@ where
             .flat_map(|(id, dec_state)| dec_state.plaintext().map(|pt| (id.clone(), pt)))
             .collect();
         if !proposer_ids.iter().eq(plaintexts.keys()) {
+            debug!(
+                "{:?} Cannot output batch. Some accepted contributions are not decrypted yet: {:?}",
+                self.netinfo.our_id(),
+                {
+                    let decrypted: BTreeSet<N> = plaintexts.keys().cloned().collect();
+                    let diff: Vec<N> = proposer_ids.difference(&decrypted).cloned().collect();
+                    diff
+                }
+            );
```

Comment:
That's probably also very noisy.
(I guess most of the `debug!` logs in this PR are not meant to get merged eventually?)